kernel/init.c

```diff
@@ -395,6 +396,9 @@ void z_init_cpu(int id)
 	_kernel.cpus[id].usage.track_usage =
 		CONFIG_SCHED_THREAD_USAGE_AUTO_ENABLE;
 #endif
+
+	/* Increment number of CPUs initialized on PM subsys */
+	pm_cpus_active_inc();
```

Comment:
Naming: "increment" is an implementation detail; the kernel code here doesn't care that the PM subsystem has a count of CPUs.  Maybe call this "pm_cpu_init()" or "pm_cpu_register()" instead, implying that this is the spot where the power management system gets to set up the CPU details however it likes?

Reply:
yep, that makes sense. My initial implementation was using `pm_cpu_active_set`. `pm_cpu_init()` is better imho.